lib/_http_agent.js

```diff
@@ -336,18 +336,18 @@ Agent.prototype.reuseSocket = function reuseSocket(socket, req) {
 };
 
 Agent.prototype.destroy = function destroy() {
-  const sets = [this.freeSockets, this.sockets];
-  for (var s = 0; s < sets.length; s++) {
-    var set = sets[s];
-    var keys = Object.keys(set);
-    for (var v = 0; v < keys.length; v++) {
-      var setName = set[keys[v]];
-      for (var n = 0; n < setName.length; n++) {
-        setName[n].destroy();
-      }
+  destroySockets(this.freeSockets);
+  destroySockets(this.sockets);
+};
+
+function destroySockets(sockets) {
+  for (const setKey in sockets) {
+    var setName = sockets[setKey];
```

Comment:
const?

Reply:
Resolved in https://github.com/nodejs/node/pull/28596/commits/3d0ec309fc15619fe7c1635626659a22d9b2aaa8